src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -20,7 +20,7 @@ public abstract class DeleteCommand<T> extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the entity identified by subcommand 'contact', 'company' or 'job' and "
             + "the index number used in the displayed person list.\n"
-            + "Parameters: [contact/job/company] INDEX (must be a positive integer)\n"
+            + "Parameters: [contact/job/company] INDEX (must be a positive integer between 1 and 2147483647)\n"
```

Comment:
Understood, keep this for now, I will do the necessary changes in a later PR regarding command validation.